e2e/test/scenarios/collections/collections.cy.spec.js

```diff
@@ -774,13 +774,10 @@ describe("scenarios > collection items listing", () => {
     cy.findByTestId(testId).findByText(columnName).click();
   }
 
-  function getAllCollectionItemNames() {
```

Comment:
`getAllCollectionItemNames` was breaking the chain in Cypress, so assertions did not know how to wait for DOM to be updated